tests/wpt/mozilla/tests/mozilla/response-data-brotli.htm

```diff
@@ -9,34 +9,27 @@
   <body>
     <div id="log"></div>
     <script>
-      function request(input) {
-        var test = async_test();
-        test.step(function() {
-          var client = new XMLHttpRequest()
-
-          client.open("POST", "resources/brotli.py", false);
-
-          client.onreadystatechange = test.step_func(function () {
-              if (client.readyState === 4) {
-                  var len = parseInt(client.getResponseHeader('content-length'), 10);
-
-                  assert_equals(client.getResponseHeader('content-encoding'), 'br');
-                  assert_true(len < input.length);
-                  assert_equals(client.responseText, input);
-                  test.done();
-              }
+      async_test(function(test) {
```

Comment:
Nit: you could do `var test = async_test();` and not have to indent the entire test.

Reply:
That's how the test was originally written, but that excludes the code outside of the test from reporting errors. This format provides better coverage.